pkg/metadata/host/host.go

```diff
@@ -152,6 +152,24 @@ func getHostAliases() []string {
 	return aliases
 }
 
+// try to get the public IPv4 address using AWS and GCE
+func getPublicIPv4() (string, error) {
+	publicIPv4, err := ec2.GetPublicIPv4()
+	if err == nil {
+		log.Infof("EC2 public IP = %s", publicIPv4)
+		return publicIPv4, err
+	}
+
+	publicIPv4, err = gce.GetPublicIPv4()
+	if err == nil {
+		log.Infof("GCE public IP = %s", publicIPv4)
```

Comment:
Is `Info` the appropriate level for these logs? On success, there would be two `Info`-level logs.

Reply:
I'm a sucker for chatty logs, but if you think just one would be sufficient I can change it :)

Reply:
Unless you mean the info logs for EC2 and GCE, in which case you will only ever get 1 info log

Reply:
I changed them to Debug, makes sense